Разметка всех страниц

Inga-e opened the pull request:
Сделала разметку

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2166645/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/2166645-device-35/2/

Inga-e committed:
Делает header

Inga-e committed:
/footer и часть header

Inga-e committed:
Разметка в файлах index catalog

Inga-e committed:
Добавляет плюсы в файл readme

Inga-e committed:
Делает плюсики в readme

проверка связи

Inga-e committed:
Убирает лишнюю строку

Строка была пустая, убрала для проверки связи для создания пулл-реквест

Inga-e committed:
Возвращает предыдущий вид файла

Вносила изменения и вернула назад

saimon395 commented on catalog.html:
```diff
@@ -0,0 +1,288 @@
+      <section class="search-section">
+        <h2 class="visually-hidden">Поиск</h2>
+        <form class="search-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label for="search"></label>
+          <input class="field" placeholder="Поиск по сайту" type="text" name="search" id="search">
```
просто лэйбл без описания не особо нужен, добавь туда какой нибудь текст и класс visually-hidden

Inga-e replied:
Сделала

saimon395 commented on catalog.html:
```diff
@@ -0,0 +1,288 @@
+        <form class="search-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label for="search"></label>
+          <input class="field" placeholder="Поиск по сайту" type="text" name="search" id="search">
+          <button class="button search-button" type="button">
+            <span class="search-button">Найти</span>
+          </button>
```
у кнопки в орме тип submit должен быть

Inga-e replied:
Тоже сначала поставила submit, потом засомневалась и убрала.
Сделала.

saimon395 commented on index.html:
```diff
@@ -4,9 +4,286 @@
+      </form>
+    </section>
+    <ul class="main-navigation-list-user">
+      <li class="main-navigation-user-item">
+        <a class="main-navigation-user-link main-navigation-user-in" href="#">
+          Войти
+          </a>
+      </li>
```
списки с одним элементом не создавай, здесь просто див используй

Inga-e replied:
Сделано

saimon395 commented on index.html:
```diff
@@ -4,9 +4,286 @@
+        <button class="main-navigation-user-button" type="button">Корзина</button>
+      </li>
+    </ul>
+    <ul lass="main-navigation-list" >
+      <li class="main-navigation-item">
+        <a class="main-navigation-link" href="#">Каталог товаров</a>
+        <button class="main-navigation-button" type="button">
+          +
+        </button>
+      </li>
+    </ul>
+    <ul class="navigation-list">
+      <li class="navigation-item">
+        <a class="navigation-link" href="#">Доставка</a>
+      </li>
+      <li class="navigation-item">
+        <a class="navigation-link" href="#">Гарантия</a>
+      </li>
+      <li class="navigation-item">
+        <a class="navigation-link" href="#">Контакты</a>
+      </li>
+    </ul>
+    </nav>
```
что то тут явно пунктов меню не хватает, в первый li добавь список пунктов , которые при открытом меню появляются

Inga-e replied:
Я их сначала добавила, потом убрала. В задании написано, что пока не надо заполнять то, что выпадает

Inga-e replied:
сделано

saimon395 commented on index.html:
```diff
@@ -4,9 +4,286 @@
+            </li>
+            <li class="slider-pagination-item">
+              <button class="slider-pagination-button slider-pagination-button-current" type="button">
+                <span class="visually-hidden">2 фото</span>
+                <b class="slider-title"></b>
+                <p class="slider-description"></p>
+                <a class="slider-link" href="#">Подробнее</a>
+                <dl class="slider-list">
+                  <dt class="slider-term"></dt>
+                  <dd class="slider-definition"></dd>
+                  <dt class="slider-term"></dt>
+                  <dd class="slider-definition"></dd>
+               </dl>
+             </button>
+            </li>
+            <li class="slider-pagination-item">
+             <button class="slider-pagination-button slider-pagination-button-current" type="button">
```
описание остальным слайдам тоже добавь

Inga-e replied:
Сделано

saimon395 commented on index.html:
```diff
@@ -4,9 +4,286 @@
+
+      </section>
+
+      <div class="suggestion">
+
+         <ul class="suggestion-list">
+             <li class="suggestion-item">Доставка</li>
+             <li class="suggestion-item">Гарантия</li>
+             <li class="suggestion-item">Кредит</li>
+         </ul>
+             <img class="vertical-line" src="#" alt="Вертикальная полоса">
+      </div>
+
+      <section class="delivery">
+
+        <h2 class="delivery">Доставка</h2>
+        <p class="delivery-description">Мы с удовольствием доставим ваш товар прямо к вашему подъезду совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его на ваш этаж.</p>
+        <img class="moving man" src="#" alt="Движущийся человек">
+
+      </section>
```
вот это по сути одна секция, просто там несколько вкладок которые можно переключать, так что помимо доставки сделай и остальные

Inga-e replied:
Сначала тоже сделала одной секцией, потом подумала, что надо разделить :)

Inga-e replied:
Сделано

saimon395 commented on index.html:
```diff
@@ -4,9 +4,286 @@
+      </ul>
+      <ul class="main-navigation-list-user">
+      <li class="main-navigation-user-item">
+        <button class="main-navigation-user-button" type="button">Сравнить</button>
+      </li>
+      <li class="main-navigation-user-item">
+        <span class="basket-set">2</span>
+        <button class="main-navigation-user-button" type="button">Корзина</button>
+      </li>
```
Сравнить и Корзина делай ссылками

Inga-e replied:
Сделано

saimon395 commented on index.html:
```diff
@@ -4,9 +4,286 @@
+      </li>
+    </ul>
+    <ul lass="main-navigation-list" >
+      <li class="main-navigation-item">
+        <a class="main-navigation-link" href="#">Каталог товаров</a>
```
вот здесь ссылку на страницу каталога добавь что бы можно было со страницы на страницу ходить

Inga-e replied:
А, точно, это просмотрела

Inga-e replied:
Сделано

saimon395 commented on catalog.html:
```diff
@@ -0,0 +1,288 @@
+      </li>
+      </ul>
+
+      <button class="button pagination-show-more" type="button">Загрузить ещё</button>
+
+      <ul class="pagination">
```
это тоже сылкой делай

Inga-e replied:
Сделала

Inga-e committed:
Исправляет catalog

1. добавила текст и класс "visually-hidden" в label по поиску
2. в форме кнопку поиска сделала с типом  submit
3. сделала ссылку на загрузить еще

Inga-e committed:
Merge branch 'master' of https://github.com/Inga-e/2166645-device-35

Inga-e committed:
Добавляет 2 корректировки в каталог

Вот здесь 2 исправления в каталоге:
1. тип кнопки на submit
2. ссылка вместо кнопки на загрузить еще

Inga-e committed:
Изменяет index

Внесла все исправления по замечаниям
